new-frontiers-14/frontier-station-14
# Apothecary Shuttle Again!

This is the Apothecary release candidate that received the Magnus walkthrough. Never again.

Because nobody makes a fool out of Dusty Lens 7 times.




## About the PR



Shuttle, guidebook, data for the NM Apothecary. Reuploaded to a new branch as the last one was saturated with undesired files.


## Why / Balance


The ship seeks to fill a gap in the lineup of medical craft and to provide the player base with a medic specific craft capable of fabricating basic medicines without the need to pull a much larger hospital sized ship.

## How to test


Check the guidebook entry.
Purchase ship. Ensure 30,000 price.
Test fly. Open cabinets. Check engines on. Check shutters. Check blast doors. Check atmos.

## Media


![apothecary](https://github.com/new-frontiers-14/frontier-station-14/assets/54123313/eedf52fb-226e-45fd-a725-d4ecc19712c5)


- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes


None. My mental health cannot accept any.

**Changelog**
